sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -160,19 +158,16 @@ export interface MetricNamespaceName {
 }
 
 // @public (undocumented)
-export class MetricsClient {
+export interface MetricsClientOptions extends PipelineOptions {
```

Comment:
Friendly note that this is undocumented 😄

Reply:
Fixed!